projects/aca-playwright-shared/src/page-objects/components/actions-dropdown.component.ts

```diff
@@ -50,9 +50,10 @@ export enum ActionType {
 }
 
 export class ActionsDropdownComponent extends BaseComponent {
-  private static rootElement = 'aca-rule-action-list';
+  private static rootElement = 'mat-dialog-container aca-rule-action-list';
 
-  private getOptionLocator = (optionName: string): Locator => this.page.locator('.mat-select-panel .mat-option-text', { hasText: optionName }).first();
+  private getOptionLocator = (optionName: string): Locator =>
+    this.page.locator('.mat-select-panel .mat-option-text', { hasText: optionName }).first();
```

Comment:
we should introduce test ids for these components so that tests rely on those instead of material selectors

Reply:
I will check that but many locators are missing test-automation-id and adding them is not a simple process sadly

Reply:
This locator will be updated in https://hyland.atlassian.net/browse/ACS-8369